src/styles/colour/index.md.njk

```diff
@@ -59,11 +62,12 @@ directly rather than reusing `$govuk-error-colour`.
 Use these colours for graphs and supporting material.
 
 To reference colours from the palette directly you should use the `govuk-colour`
-function.
+function, for example `color: govuk-colour("blue")`.
 
-Avoid using the palette colours directly when there is a contextual colour that
-meets your use case. For example, if you are styling a component's error state
-you should use `$govuk-error-colour` rather than `govuk-colour("red")`.
+Avoid using the palette colours if there is a Sass variable that is designed
+for your context. For example. For example, if you are styling the error state
```

Comment:
@36degrees There is an extra 'For example.' in this line.

Reply:
😬